docs/src/main/asciidoc/_elytron/Credential_Store.adoc

```diff
@@ -235,6 +235,80 @@ ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMfncZuH
 cZrYRyy9+X68fylEWBMzyg+nhjWkxJlJ2M2LAGY=
 ----
 
+===== Management Operation
+
+Using a management operation the following command will generate and store a new key pair credential under the alias `example` using the ecdsa algorithm with 256 key size:
```

Comment:
Very minor comment, just for better readability would be good to update ecdsa to ECDSA.